services/osslifecycle/osslifecycle.service.js

```diff
@@ -9,49 +11,33 @@ single line similar to the following: \`osslifecycle=active\`. Other suggested v
 can be viewed on the [OSS Tracker repository](https://github.com/Netflix/osstracker).
```

Comment:
I can't leave a comment on the exact line because it hasn't been changed in the diff, but can we remove "GitHub" from the docs here

```diff
- simply create an OSSMETADATA tagging file at the root of your GitHub repository containing a
+ simply create an OSSMETADATA tagging file at the root of your repository containing a
```